databuilder/transformer/sql_to_table_col_usage_transformer.py

```diff
@@ -1,5 +1,7 @@
+import copy_reg
```

Comment:
it seems the module gets renamed in py3(https://stackoverflow.com/questions/49044133/no-module-named-copy-reg-in-python3)

Reply:
Good catch!